lib/sioFb.js

```diff
@@ -13,6 +13,14 @@ const PHOTOS_CONFIG = {
 // poll every 5 minutes
 const PHOTOS_POLLING = 1000 * 60 * 5;
 
+const getLatestStripsAfter = function(lastPolledDate) {
+  console.log('Polling last run on %s.', lastPolledDate);
+  return this.getLatestStrips()
+    .filter(x => {
+      return (lastPolledDate === null || x.created_time > lastPolledDate);
```

Comment:
Of course this works only because of https://github.com/lecoselol/node-telegram-sio-bot/pull/2/commits/f44d04c2b14f60d39f8412191181c0481d6a7cd9#diff-0f2305cda36e5ca1efec451070815ae4R52.

Many lols